README.md

```diff
@@ -25,9 +25,6 @@ clone repo: `git clone https://github.com/netman5/react-calculator.git`
 then
 `cd calculator`
 
-then
-`cd calculator-app`
-
 ### Install
 
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_ on line 34, as this project is done solely. This will make your ReadMe appear more professional.

Reply:
Thank you @rloterh  for the suggestion.